hermes_bridge/__main__.py

```
@@ -440,24 +464,32 @@ def _proxy_create(client_kwargs, *, reason, shared):
-        register_belayer_tools(
-            agent,
-            agent_id,
-            session_id,
-            socket_path,
-            allowed_tools=allowed_tools,
-            agent_kind=agent_kind,
-            turn_mail_ids=turn_mail_ids,
-        )
+        from hermes_plugins import belayer as belayer_plugin  # type: ignore[import]
+        pop_plugin_mail_ids = belayer_plugin.pop_turn_mail_ids
     except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)